opensrp-tasking/src/main/java/org/smartregister/tasking/util/TaskingMapHelper.java

```diff
@@ -0,0 +1,232 @@
+                    indexCaseSource.setGeoJson(circleFeature);
+                } else { // Clear outer circle if there is no index case
+                    if (indexCaseSource != null)
+                        indexCaseSource.setGeoJson(FeatureCollection.fromFeatures(new ArrayList<>()));
+                }
+            }
+        } catch (JSONException e) {
+            Timber.e(e);
+        }
+    }
+
+    public Feature getIndexCase(FeatureCollection featureCollection) {
+        Feature indexCase = null;
+        for (Feature feature : featureCollection.features()) {
+            if (feature.hasProperty(IS_INDEX_CASE) && feature.getBooleanProperty(IS_INDEX_CASE)) {
+                indexCase = feature;
+                break; // index case already found, no need to proceed
+            }
+        }
+        return indexCase;
+    }
+
+    public LineLayer getIndexCaseLineLayer() {
+        return indexCaseLineLayer;
+    }
+
+    public static void addOutOfBoundaryMask(@NonNull Style mMapboxMapStyle, Feature operationalArea, Feature boundingBoxPolygon, Context context) {
+        // create multi polygon
+        List<Polygon> polygonList = new ArrayList<>();
+        polygonList.add((Polygon) boundingBoxPolygon.geometry());
+        if (operationalArea.geometry() instanceof MultiPolygon) {
+            polygonList.addAll(((MultiPolygon) operationalArea.geometry()).polygons());
+        } else {
+            polygonList.add((Polygon) operationalArea.geometry());
+        }
+
+        MultiPolygon opAreaMultiPolygon = MultiPolygon.fromPolygons(polygonList);
+
+        // create mask source
+        GeoJsonSource outOfBoundarySource = new GeoJsonSource(OUT_OF_BOUNDARY_SOURCE, opAreaMultiPolygon);
+        mMapboxMapStyle.addSource(outOfBoundarySource);
+
+        // add mask
+        FillLayer maskLayer = new FillLayer(OUT_OF_BOUNDARY_LAYER, outOfBoundarySource.getId());
+        maskLayer.withProperties(fillColor(context.getResources().getColor(R.color.outside_area_mask)),
+                fillOpacity(OUTSIDE_OPERATIONAL_AREA_MASK_OPACITY));
+        mMapboxMapStyle.addLayer(maskLayer);
+
+    }
+
+    public boolean isMyLocationComponentActive(Context context, ImageButton myLocationButton) {
+        return context.getResources().getDrawable(R.drawable.ic_cross_hair_blue).getConstantState().equals(myLocationButton.getDrawable().getConstantState());
+    }
+
+    public void addBaseLayers(KujakuMapView kujakuMapView, Style style, Context context) {
+        BaseLayerSwitcherPlugin baseLayerSwitcherPlugin = new BaseLayerSwitcherPlugin(kujakuMapView, style);
+        DigitalGlobeLayer digitalGlobeLayer = TaskingLibrary.getInstance().getTaskingLibraryConfiguration().getDigitalGlobeLayer();
+        MapBoxLayer mapBoxLayer = new MapBoxLayer();
+        baseLayerSwitcherPlugin.addBaseLayer(digitalGlobeLayer, true);
```

Comment:
I was having an issue with this. We are not using DigitalGlobe and that do not have access to this plugin or the option to use mapbox as the default base layer. Can we either fix it or schedule to fix it later?